sigal/themes/colorbox/templates/index.html

```diff
@@ -120,7 +120,7 @@
               <div style='display:none'>
                 <div id="{{ media.filename|replace('.', '')|replace(' ', '') }}">
                   <video controls>
-                  <source src='{{ media.filename }}' type='video/webm' />
+                  <source src='{{ media.filename }}' {% if not settings.use_orig %}type='video/webm'{% endif %}/>
```

Comment:
I would be easy to add the type as an attribute of the Video object, based on the file extension, so it is probably better to do it just in case.